File: src/main/java/com/github/wgzhao/dbquery/entities/Sign.java
package com.github.wgzhao.dbquery.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name="signs")
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Sign
{
    @Id
    private String appId;

    private String appKey;

    private String applier;
}

File: src/main/kotlin/com/github/wgzhao/dbquery/entities/Sign.kt
package com.github.wgzhao.dbquery.entities

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import java.sql.Timestamp

@Entity
@Table(name = "signs")
data class Sign(
    @Id
    var appId: String,
    var appKey: String,
    var applier: String? = null,
    var enabled: Boolean = false,
    val createdAt: Timestamp = Timestamp(System.currentTimeMillis()),
    val updatedAt: Timestamp = Timestamp(System.currentTimeMillis())
)
